test/distributed/tensor/test_attention.py

```diff
@@ -41,6 +42,8 @@
     backends.append(SDPBackend.FLASH_ATTENTION)
 if PLATFORM_SUPPORTS_MEM_EFF_ATTENTION:
     backends.append(SDPBackend.EFFICIENT_ATTENTION)
+if PLATFORM_SUPPORTS_CUDNN_ATTENTION:
```

Comment:
We will be recommending only using this option on h100 and b200 btw

Reply:
Will `PLATFORM_SUPPORTS_CUDNN_ATTENTION` reflect this? if so, we may need to manually copy the existing `PLATFORM_SUPPORTS_CUDNN_ATTENTION` check into the test since we do want to run this unit test on CI instance.